code/index.html

```diff
@@ -1,17 +1,208 @@
+      rel="stylesheet"
+    />
+    <link rel="icon" href="./assets/favicon.svg" />
+    <link rel="mask-icon" href="./assets/favicon.svg" color="#000000" />
+    <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <h1>Business name 🌻</h1>
-
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
+    <header class="page-header">
+      <a href="./index.html" class="page-header__logo">
+        <img
+          width="120"
+          height="35"
+          src="./assets/picko-logo.svg"
+          alt="Picko's logotype"
+        />
+      </a>
+      <nav class="page-header__nav">
+        <a href="#nav" class="page-header__nav__link">
+          <div class="hamburger"></div>
+          <span class="text">Menu</span>
+        </a>
+        <div class="page-header__menu">
+          <ul>
+            <li><a href="#link1">Home</a></li>
+            <li><a href="#link1">Open ads</a></li>
+            <li><a href="#link1">About us</a></li>
+            <li><a href="#link1">Contact</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="hero hero--video">
+        <div class="hero__content">
+          <h1 class="hero__title">When you like to move it, move it</h1>
+          <p class="hero__text text-large">
+            Picko is your go-to platform for easy item transport and delivery.
+            Instantly connect with reliable helpers to get your goods from point
+            A to point B.
+          </p>
+          <form
+            action="https://httpbin.org/anything"
+            method="post"
+            class="hero__form"
+          >
+            <label>
+              <span>First name</span>
+              <input
+                type="text"
+                name="first_name"
+                autocomplete="given-name"
+                placeholder="Madonna"
+                required
+              />
+            </label>
+            <label>
+              <span>Email</span>
+              <input
+                type="email"
+                name="email"
+                autocomplete="email"
+                placeholder="your@email.com"
+                required
+              />
+            </label>
+            <div class="hero__form__button">
+              <button type="submit">
+                <span class="text">Get started</span
+                ><span class="icon"
+                  ><img
+                    src="./assets/arrow-forward.svg"
+                    alt="Arrow going forward"
+                /></span>
+              </button>
+            </div>
+          </form>
+        </div>
+        <div class="hero__video bg-video-wrapper">
+          <video
+            muted
+            playsinline
+            autoplay
+            loop
+            preload="auto"
+            poster="./assets/video-poster.jpg"
+          >
```

Comment:
The use of a video background adds a cool touch to the hero section. You have used included in video element attributes (muted, plays inline, autoplay, loop) for smooth functionality - but I read that maybe a controls attribute might be a good fallback option in case autoplay fails.

Reply:
Thank you Kelly, I'll look into that 🙏